### ISSUE TITLE
Unable to reproduce the results for ACM. Results highly depend on random seed.

### ISSUE THREAD

**Issue by gau-nernst** [liun-online/HeCo#10]
Hello!

Using the default hyperparameters and the datasets provided in this repo, I am unable to reproduce the results for ACM. F1 macro, F1 micro and AUC are consistently around 1-2% worse than the results published in the paper.

I have also discovered that results highly depend on the random seed. Sometimes, a good seed can give very good results, while a bad seed can be -2% or -5%. This is only observed in ACM dataset, and to a smaller extent in AMiner dataset. For DBLP and Freebase, the results don't vary as much with random seed. I wonder if you observe the same problem?

For the reported results, `randomly run 10 times and report the average results`, did you run the experiments 10 times with different seeds? I saw that the seed for each dataset is fixed in `params.py`. Did you tune the seed for each dataset?

Thank you

**Comment by liun-online** [comment]
Thanks for your attention!
1. We didn't notice the issue you mentioned. Actually, we just perform the same experimental operations on ACM compared with other datasets. The instability observed in ACM may relate to the characteristics of ACM, or used versions of different packages (e.g., PyTorch), or CPU / GPU. To be honest, we actually didn't meet the same problem. You can see that the reported results on ACM have smaller standard deviations compared with Freebase and AMiner.
2. About fixed seed. When doing experiments, we just did as we stated "randomly run 10 times and report the average results". In this case, we didn't tune the seed. However, when opening our source code, we should make the users convinced that we can reach the reported results. So, in this process, we choose a case where the results are better than the reported ones.
